scene/2d/polygon_2d.cpp

```diff
@@ -101,7 +101,11 @@ void Polygon2D::_skeleton_bone_setup_changed() {
 }
 
 void Polygon2D::_notification(int p_what) {
+	if (p_what == NOTIFICATION_TRANSFORM_CHANGED && !Engine::get_singleton()->is_editor_hint())
+		return; // Mesh recreation for NOTIFICATION_TRANSFORM_CHANGED is only needed in editor.
+
```

Comment:
```suggestion
	if (p_what == NOTIFICATION_TRANSFORM_CHANGED && !Engine::get_singleton()->is_editor_hint()) {
		return; // Mesh recreation for NOTIFICATION_TRANSFORM_CHANGED is only needed in editor.
	}

```

This is just a style suggestion. We always use curly braces with if statements